File: src/main/java/ru/testproject/catalog/to/CategoryTO.java
package ru.testproject.catalog.to;

import javax.validation.constraints.NotBlank;

import java.io.Serializable;

public class CategoryTO implements Serializable {
    @NotBlank(message = "Category name is mandatory")
    public String name;
    @NotBlank(message = "New category name is mandatory")
    public String newCatName;
}

File: src/main/kotlin/ru/petproject/catalog/to/CategoryTO.kt
package ru.petproject.catalog.to

import jakarta.validation.constraints.NotBlank
import java.io.Serializable

class CategoryTO : Serializable {
    var name: @NotBlank(message = "Category name is mandatory") String? = null
    var newCatName: @NotBlank(message = "New category name is mandatory") String? = null
}
